package.json

```diff
@@ -24,27 +24,41 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build admin --configuration production && ng build public-search --configuration production",
-    "test": "ng test admin && ng test public-search",
-    "lint": "ng lint admin && ng lint public-search",
-    "e2e": "ng e2e admin && ng e2e public-search"
+    "test": "ng test",
```

Comment:
Is these tests run the two projects?

Reply:
yes, both tests are running with "ng test" command